unit-9/exercise-5.js

```diff
@@ -0,0 +1,16 @@
+// a, b lần lượt là điểm thi giữa kỳ, điểm thi cuối kỳ
+const ranks = (a, b) => {
+  const medium = (a + b )/2
+  if (medium >= 9.0) {
+    return 'Hạng A'
+  }
+  if ( 7.0 <= medium && medium < 9.0) {
```

Comment:
```suggestion
  if (7.0 <= medium && medium < 9.0) {
```
Bỏ `.0` đi

Hoặc nếu viết theo chuỗi `if else` thì đoạn trên viết thành

```suggestion
  if (average >= 9) return 'A'
  else if (average > 7) return 'B'
```